src/Mantasflowers.WebApi/appsettings.json

```diff
@@ -15,5 +15,9 @@
   "Githash": "#{__Githash}#",
   "FirebaseConfig": "",
   "ProjectId": "",
-  "WebApiKey": ""
+  "WebApiKey": "",
+  "Stripe": {
+    "SecretKey": "",
+    "PublishableKey": ""
+  }
```

Comment:
Do we have different values for Azure? Or are they the same?

Reply:
Would prefer to discuss it in private.

Reply:
Now that I think about it, unless frontend would manage publishable key in the same way, test keys will have same values.